documentation/tutorial.md

```diff
@@ -68,3 +71,622 @@ tutorial overlord    localhost/localhost  2.9.37   unsupported  23:20:53Z
 Model "admin/tutorial" is empty.
 ```
 
+
+## Deploying Charmed MongoDB:
+To deploy Charmed MongoDB, fetch the charm from [Charmhub](https://charmhub.io/mongodb?channel=dpe/edge) and deploy it to your model:
+```
+juju deploy mongodb --channel dpe/edge
+```
+
+Juju will now fetch Charmed MongoDB and begin deploying it to the LXD cloud. This process can take several minutes depending on how provisioned your machine is. You can track the progress by running:
+```
+watch -c juju status --color
+```
+
+This will show charmed MongoDB along with its current status and other helpful information.
+
+Wait until the application is ready, when it is ready `watch -c juju status --color` will show
+```
+Every 2.0s: juju status --color                                                                                         ip-172-31-11-104: Fri Dec  2 11:24:30 2022
+
+Model     Controller  Cloud/Region         Version  SLA          Timestamp
+tutorial  overlord    localhost/localhost  2.9.37   unsupported  11:24:30Z
+
+App      Version  Status  Scale  Charm    Channel   Rev  Exposed  Message
+mongodb           active      1  mongodb  dpe/edge   96  no
+
+Unit        Workload  Agent  Machine  Public address  Ports      Message
+mongodb/0*  active    idle   0        10.23.62.156    27017/tcp
+
+Machine  State    Address       Inst id        Series  AZ  Message
+0        started  10.23.62.156  juju-d35d30-0  focal       Running
+```
+
+## Accessing MongoDB:
+*Disclaimer: this part of the tutorial accesses MongoDB via the `admin` user. It is not recommended that you directly interface with the admin user in a production environment. In a production environment [create a separate user](https://www.mongodb.com/docs/manual/tutorial/create-users/) and connect to MongoDB with that user instead. Later in the section covering Relations we will cover how to access MongoDB without the admin user.
```

Comment:
maybe call this out as a proper admonition

Reply:
What is the best way to " call this out as a proper admonition"? I changed the wording to be more severe, does that help? Or is there something I should explicitly say.

Reply:
yup, thats what I meant! sorry didn't see your note before